src/snowflake/snowpark/_internal/analyzer/expression.py

```diff
@@ -80,6 +96,18 @@ def sql(self) -> str:
         )
         return f"{self.pretty_name}({children_sql})"
 
+    @property
```

Comment:
Can we add some comments here or in complexity_stat.py to brief talk about the main idea of complexity score calculation (e.g., use # of columns involved as a proxy, etc.)?

Reply:
added